charts/yurt-manager/crds/apiextensions.k8s.io_v1_customresourcedefinition_yurtappoverriders.apps.openyurt.io.yaml

```diff
@@ -0,0 +1,144 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
please rename this file name into `apps.openyurt.io_yurtappoverriders.yaml`

Reply:
fix